### PR TITLE
Update the theme

### PR DESCRIPTION
**Pull request by hellcp**
This updates the theme to the newer version of chameleon on openSUSE side of things. I also added a simple bit of js to show a tooltip after copying an image command, so it should be a bit more reactive and easy to tell that it was copied

I could also add megamenu (the menu linking to other openSUSE website accessible in the top right of most of our websites, like https://news.opensuse.org), but I don't know if you would be interested in that

### CODE REVIEW

File: templates/navbar.tt2 [hunk 1]
```diff
@@ -1,7 +1,7 @@
     <nav class='navbar navbar-expand-lg'>
       <a class='navbar-brand' href='https://opensuse.org' title='openSUSE'>
         [% IF theme.logo %]
-        <img class="mr-2" src="[% theme.logo %]" alt="[% theme.vendor %]"/>
+        <img class="mr-2" height="30" src="[% theme.logo %]" alt="[% theme.vendor %]"/>
```
Comment (M0ses) [inline templates/navbar.tt2:4]: The fixed height might be a problem in other themes - either make it configurable or better use a css class or id which we can define in the theme

File: templates/details.tt2 [hunk 2]
```diff
@@ -21,7 +21,7 @@
   [% uniquetag_one = name | remove('[^A-Za-z0-9]+') %]
   [% uniquetag_two = tag | remove('[^A-Za-z0-9]+') %]
   [% uniquetag = "$uniquetag_one$uniquetag_two" %]
-  <div href="#" class='btn btn-light rounded-0 card-header d-sm-flex justify-content-between align-items-center' data-toggle="collapse" data-target="#collapse[% uniquetag %]" aria-expanded="false" aria-controls="collapse[% uniquetag %]">
+  <div href="#" class='card-header d-sm-flex justify-content-between align-items-center' data-toggle="collapse" data-target="#collapse[% uniquetag %]" aria-expanded="false" aria-controls="collapse[% uniquetag %]">
```
Comment (M0ses) [inline templates/details.tt2:24]: Have you checked if this works well with the "suse" theme?

**Review by M0ses — CHANGES_REQUESTED** [review]

File: templates/details.tt2 (hunk 2 above)
Comment (hellcp) [reply templates/details.tt2:24]: It does, in fact it fixes some issues with the theme too

Comment (M0ses) [reply templates/details.tt2:24]: Nice!

**Comment by M0ses** [comment]
THX for your contribution

**Pull request merged**